# Experiment f8abdd6f2faab6333c82ab76671b98b7692c208238b2e100c0f44da0e8d40ecc
# Base c445e4e2dffd345dcca2845b420c2f5a7797fb58
diff --git a/src/t_b_obj.cpp b/src/t_b_obj.cpp
index a33c72b0e..2776c82e4 100644
--- a/src/t_b_obj.cpp
+++ b/src/t_b_obj.cpp
@@ -635,9 +635,9 @@ void TRIBE_Building_Object::lay_down_passable_terrain() {
     }
 
     short tx1 = (short)(world_x - object_master->radius_x);
-    short tx2 = (short)(world_x + object_master->radius_x - 10.0f);
+    short tx2 = (short)(world_x + object_master->radius_x - 0.01f);
     short y1 = (short)(world_y - object_master->radius_y);
-    short y2 = (short)(world_y + object_master->radius_y - 10.0f);
+    short y2 = (short)(world_y + object_master->radius_y - 0.01f);
     if (tx1 < 0) tx1 = 0;
     if (y1 < 0) y1 = 0;
     if (tx2 >= map->map_width) tx2 = (short)map->map_width - 1;
